esss_fix_format/cli.py

```diff
@@ -79,8 +87,7 @@ def fix_whitespace(old_contents):
     :return unicode:
         Returns the new contents.
     """
-    input_lines = old_contents.split('\n')
-    lines = [i.rstrip('\r') for i in input_lines]
+    lines = old_contents.split('\n')
```

Comment:
Actually it needs to be ```'\n'```
io.open uses ```universal newlines``` by default

But that raises another problem, when writing the new contents ```\n``` will always be converted to ```os.linesep```. But some files (e.g. *.bat) will always contain ```\r\n``` even on Linux and some (e.g. *.sh) will always contain ```\n``` even on Windows.

So we should just keep whatever line ending the file already has. This means disabling ```universal_newlines``` on ```io.open``` and using ```str.splitlines()```, which deals with any line ending. Also when joining the lines again we need to decide which line ending to use. I would say to just use the line ending of the first line